pollbot/models/poll.py

```diff
@@ -152,9 +152,9 @@ def get_date_option(self, check_date: date) -> Optional[Option]:
     def get_formatted_due_date(self) -> str:
         """Get the formatted date."""
         if self.european_date_format:
-            return self.due_date.strftime("%d.%m.%Y %H:%M UTC")
+            return self.due_date.strftime("%d.%m.%Y %H:%M")
```

Comment:
I think is this case, the time displayed is in the corresponding unit time of the poll, it is not UTC, isn't it?

Reply:
All datetimes on this codebase are actually `timezone without time zone`.
Since the server has a UTC time, this is UTC.

The thing is, that telegram usually sends UTC timestamps, and it's extremely hard to reliably know where a user is from.

Due to this, I went for the easiest approach, i.e. working with UTC.

Reply:
Sorry, I didn't express it correct. Let me give you more information.
You are right, all date times on the codebase are in UTC format, but the date time displayed on the due date is not in UTC format, it displays the date time with the location of the telegram's user. Let me give you an example:
In my country, we use `UTC -3` format, so now in my country it's 19:15 HS `UTC-3` so in `UTC` time it's ` 22:15 hs`.
If I create a poll with 30 minutes on due time duration, the due date time will be: `19:45 hs on UTC-3` and `22:45 hs on UTC`
The message displayed on my telegram is:
![image](https://user-images.githubusercontent.com/53308354/199846489-3e36ed14-66dc-43ad-8f43-da28af68749d.png)
If you see the picture, the date time displayed is `UTC-3` (19:45) and not `UTC` (22:45)
So that why I think that we use remove the text `UTC`.
Another thing to mention, I don't if it's right, but the due date time in the database is saved on `UTC -3` and created at is saved in `UTC`. Here is the output:

```
pollbot=# select id, created_at, due_date from Poll;
 id |         created_at         |          due_date          
----+----------------------------+----------------------------
  2 | 2022-11-03 22:15:27.480942 | 2022-11-03 19:45:27.264481
````
The poll works fines, it's finished on my local time, but I don't if we should save the date time on UTC format like the other fields.
Let me know what do you think about the example.

Reply:
This is because you're running your database on a server/pc that has the same local time as you.
Once you run this on a server in a different timezone, you'll see the expected time shift.

The official Ultimate Pollbot server runs on a server that has a UTC timezone -> All times are UTC.

If we're going to deploy this code on the real instance, users will see a local time i.e. 19:45, even though the time will actually be 19:45 UTC.

Reply:
I admit that this is quite hacky, but back when I started this project, I didn't spend a single thought on running this for a global userbase :D  

In general, it would be best to refactor this whole system.

But for this to properly work, we would still have to keep track of user's locales, which isn't easily possible for bots (at least it wasn't last time I checked).